Ensure tests run without API keys

Currently tests fail:
```python
    def get_web3(chain_id: int) -> Web3:
        """Get a Web3 instance for the given chain ID."""
        node_url = get_infura_node_url(chain_id)
        if "None" in node_url:
>           raise ValueError("No API key found. Please set the environment variable INFURA_API_KEY.")
E           ValueError: No API key found. Please set the environment variable INFURA_API_KEY.

src/tellor_disputables/data.py:56: ValueError
=============================================== short test summary info ===============================================
FAILED tests/test_alerts.py::test_get_twilio_client - twilio.base.exceptions.TwilioException: Credentials are requir...
FAILED tests/test_data.py::test_rpc_value_errors - ValueError: No API key found. Please set the environment variable...
```
Needs to just output a warning instead

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -1,5 +1,8 @@
 import pytest
 from tellor_disputables.data import get_web3
+from twilio.base.exceptions import TwilioException
+from tellor_disputables.alerts import get_twilio_client
+import warnings
 
 
 @pytest.fixture
@@ -7,4 +10,13 @@ def check_web3_configured() -> None:
     try:
         _ = get_web3(chain_id=1)
     except ValueError as e:
+        warnings.warn(str(e))
+        pytest.skip(str(e))
+
+@pytest.fixture
+def check_twilio_configured() -> None:
+    try:
+        _ = get_twilio_client()
+    except TwilioException as e:
+        warnings.warn(str(e))
         pytest.skip(str(e))

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -1,0 +1,10 @@
+import pytest
+from tellor_disputables.data import get_web3
+
+
+@pytest.fixture
+def check_web3_configured() -> None:
+    try:
+        _ = get_web3(chain_id=1)
+    except ValueError as e:
+        pytest.skip(str(e))

--- a/tests/test_alerts.py
+++ b/tests/test_alerts.py
@@ -27,7 +27,7 @@ def test_get_from_number():
     assert num == "+19035029327"
 
 
-def test_get_twilio_client():
+def test_get_twilio_client(check_twilio_configured):
     client = get_twilio_client()
 
     assert isinstance(client, Client)

--- a/tests/test_data.py
+++ b/tests/test_data.py
@@ -86,7 +86,7 @@ def test_get_query_from_data():
 
 
 @pytest.mark.asyncio
-async def test_rpc_value_errors(caplog):
+async def test_rpc_value_errors(check_web3_configured, caplog):
 
     error_msgs = {
         "{'code': -32000, 'message': 'unknown block'}": "waiting for new blocks",

